packages/librhsp/lib/binding.ts

```diff
@@ -23,11 +22,19 @@ import {
     Version,
 } from "@rev-robotics/rev-hub-core";
 import { SerialParity } from "@rev-robotics/rev-hub-core";
+import * as path from "path";
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 const require = createRequire(import.meta.url);
-const addon = require("node-gyp-build")(path.join(__dirname, ".."));
+console.log(__dirname);
+const addon = require("pkg-prebuilds")(
+    path.join(__dirname, ".."),
+    {
```

Comment:
This needs to be synced with the binding-options.cjs file if kept like this. I was getting errors trying to require the file.